packages/react-core/src/components/TextInput/TextInput.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/FormControls/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const inputTypes = [
+  'text',
+  'date',
+  'datetime-local',
+  'email',
+  'month',
+  'number',
+  'password',
+  'search',
```

Comment:
We probably want to do our own search component.

Reply:
It is likely that any "search" component we make will use this TextInput component. So this type will be required for that.